python/hail/genetics/kinshipMatrix.py

```diff
@@ -15,6 +15,16 @@ def __init__(self, jkm):
         self._key_schema = None
         self._jkm = jkm
 
+    @classmethod
+    @record_classmethod
+    def _from_python(cls, sample_schema, bm, sample_ids, n_variants):
```

Comment:
I don't particularly like this method name. Maybe from_block_matrix or something?

Reply:
Sounds good. I think Jon has a plan for the eventual disappearance of KinshipMatrix, so I didn't put too much thought into it.